packages/flutter/lib/src/material/dialog.dart

```diff
@@ -218,6 +218,62 @@ class Dialog extends StatelessWidget {
 /// ```
 /// {@end-tool}
 ///
+/// {@tool dartpad --template=stateful_widget_scaffold_center}
+///
+/// This demo shows a method in a [State] which, when called, displays a dialog box
+/// and returns a [Future] that completes when the dialog is dismissed.
+///
+/// ```dart
+/// @override
+/// Widget build(BuildContext context) {
+///   Future<void> _showMyDialog() async {
+///     return showDialog<String>(
+///       context: context,
+///       builder: (BuildContext context) => AlertDialog(
+///         title: const Text('AlertDialog tilte'),
```

Comment:
Typo and capitalization.
```suggestion
///         title: const Text('AlertDialog Title'),
```